tools/docker/docker-compose-mac.yml

```diff
@@ -58,27 +83,41 @@ services:
       timeout: 5s
       retries: 10
     volumes:
-      - ${EDA_HOST_PODMAN_SOCKET_URL:-/run/user/501/podman/podman.sock}:/run/podman/podman.sock:z
+      - ${EDA_HOST_PODMAN_SOCKET_URL}:/run/podman/podman.sock:z
```

Comment:
This makes this environment variable mandatory for users. Is that what you want? It would be a breaking change and probably should be documented.

Reply:
@Alex-Izquierdo `EDA_HOST_PODMAN_SOCKET_URL` is defaulted in to `/run/user/501/podman/podman.sock` in `common-env` to eliminate the redundancy in its previous uses.

Reply:
I think that won't work, at least with docker-compose v1 (which we have to support) 

Common env is for the environment variables injected into the pods. Here, the scope is the docker-compose process itself, not pods. Docker-compose takes the value from the user environment. If this environment variable is not defined docker-compose would fail.

Reply:
I'd be inclined to agree if it weren't that I don't set that variable and both docker-compose (v1.29.2) and podman-compose (v1.0.6) build and run without exception that I've encountered.  If others have different experiences I'd be interested in hearing.

Reply:
Well, if it works, I'm wrong. :)